tests/logic.js

```diff
@@ -204,6 +204,33 @@ define([
                                  {load: {"/data/select": ["name", "dob"]}});
             });
         });
+
+        describe("findUsages", function () {
+            var form;
+            before(function (done) {
+                form = util.loadXML(TEST_XML_1);
+                done();
+            });
+
+            it("should return dictionary of usages", function () {
+                var expectedOutput = {
+                        "#form/question1": {"#form/question2": "Display Condition"}
+                    };
+                assert.deepEqual(form.findUsages(), expectedOutput);
```

Comment:
Maybe load a form with more usages so the expected output of this test is not the same as the one that filters on a path?